src/cli.test.ts

```diff
@@ -0,0 +1,136 @@
+import { execSync } from 'node:child_process';
+
+import { goSync } from '@api3/promise-utils';
+
+const CLI_EXECUTABLE = 'dist/src/cli.js';
+
+type CommandArg = [string, string | number | boolean];
+
+describe('cli tests', () => {
+  const execCommand = (command: string, ...args: CommandArg[]) => {
+    const quote = (val: string) => `"${val}"`;
+    const formattedArgs = args
+      .map(([c, a]) => {
+        // if args is array then quote each elem and separate them with space
+        if (Array.isArray(a)) return `${c} ${a.map((element) => quote(element)).join(' ')}`;
```

Comment:
The `CommandArg` type definition above says `a`'s type can be `string | number | boolean`. I don't see why we're handling the case where `a` is an array here.